cmd/tnf/main.go

```diff
@@ -26,6 +26,7 @@ func main() {
 	rootCmd.AddCommand(generate)
 	generate.AddCommand(catalog.NewCommand())
 	generate.AddCommand(qecoverage.QeCoverageReportCmd)
+	qecoverage.QeCoverageReportCmd.PersistentFlags().String("suitename", "", "Specify suite name.")
```

Comment:
"Specify suite name." -> "Displays the remaining tests not covered by QE for the specified suite name"